R/sampler.R

```diff
@@ -14,8 +14,13 @@
 ##' @param forecast_days Number of days to forecast forward from end
 ##'   states.  Requires that \code{save_particles} is \code{TRUE}.
 ##'
-##' @param save_particles Logical, indicating if we save full particle
-##'   histories (this is slower).
+##' @param save_particles Either 
```

Comment:
I don't love this interface.  Can you change this so that we add a new parameter:

```
save_sample_trajectory
```

or something, and if that's true create a new element in the return object